sdk/communication/communication-administration/swagger/PhoneNumber.md

```diff
@@ -19,3 +19,25 @@ payload-flattening-threshold: 10
 use: "@microsoft.azure/autorest.typescript@5.0.1"
 azure-arm: false
 ```
+
+### Rename searchId to reservationId
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions.PhoneNumberSearch.properties.searchId
+    transform: >
+      $["x-ms-client-name"] = "reservationId";
```

Comment:
It looks like this didn't work, `PhoneNumberReservation` still includes `searchId`